src/config/config.scss

```diff
@@ -140,7 +140,7 @@ $su-xxxsmall:   scaleGrid(-3) !default; // 1px
 $su-xxsmall:    scaleGrid(-2) !default; // 2px
 $su-xsmall:     scaleGrid(-1) !default; // 4px
 $su-small:       scaleGrid(1) !default; // 8px
-$su-default:     scaleGrid(2) !default; // 16px
+$su-medium:     scaleGrid(2) !default; // 16px
```

Comment:
👍 This makes more sense to me. We should probably update in Classic too.

Reply:
I can put in a follow up story for that... think there's a few other spots with the functions that need updating in classic too